pkgs/by-name/ip/ipbus-uhal/package.nix

```
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenv,
+  bash,
+  cacert,
+  boost,
+  pugixml,
+  python3,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
```

Use finalAttrs when you can
```suggestion
stdenv.mkDerivation (finalAttrs: {
```